server/file_handler.cc

```diff
@@ -9,9 +9,9 @@ RequestHandler::Status StaticFileHandler::Init(const std::string& uri_prefix, co
 	//assume the config argument has something like
 	// root /path1
 	//check if such a line exists
-	auto root_config_tokens = config.find("root");
-	if(root_config_tokens.size() < 2){
-		printf("StaticfileHandler.Init: Invalid config");
+	std::vector<std::string> root_config_tokens = config.find("root");
+	if( root_config_tokens.size() < 2 ){
```

Comment:
what I meant was 
`if (root_config_tokens.size() < 2) {`
not 
`if( root_config_tokens.size() < 2  ){`